### DISCUSSION TITLE
Adding support for FastComposer 支持 FastComposer

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#369] [MMagic]
# Adding support for FastComposer 
 支持 FastComposer 

Based on the provided open-source code at [[https://github.com/mit-han-lab/fastcomposer](https://github.com/mit-han-lab/fastcomposer)](https://github.com/mit-han-lab/fastcomposer](https://github.com/mit-han-lab/fastcomposer)) 
 1. Add support for inference

参考开源代码：[[https://github.com/mit-han-lab/fastcomposer](https://github.com/mit-han-lab/fastcomposer)](https://github.com/mit-han-lab/fastcomposer](https://github.com/mit-han-lab/fastcomposer)) 
 1. 支持推理

## Task Information

**Repository**：MMagic

**Difficulty**：Hard

**Points**：30

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：xiaomile

**Claim Time**：2023年08月09日

**PR Link**：https://github.com/open-mmlab/mmagic/pull/2011

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by xiaomile** [comment]
https://github.com/open-mmlab/mmagic/pull/2011
